macros/any_value.sql

```diff
@@ -7,7 +7,7 @@
             - Redshift
     #}
     {%- if target.type in ('postgres','redshift',) -%} 
-	max('{{val}}') 
+	MAX('{{val}}') 
     {%- elif target.type in ('snowflake','bigquery',) -%}
         any_value('{{val}}')
```

Comment:
We should probably be consistent here between targets. Since xdb is only run in test, you can lint the output of `dbt run --target snowflake ...` in addition to the postgres output